runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -88,8 +88,9 @@
 	public static final File PROFILES_DIR = new File(RUNELITE_DIR, "profiles");
 	public static final File SCREENSHOT_DIR = new File(RUNELITE_DIR, "screenshots");
 	public static final File LOGS_DIR = new File(RUNELITE_DIR, "logs");
-	public static final String DEFAULT_SESSION_FILE = "session";
-	public static final String DEFAULT_CONFIG_FILE = "settings.properties";
+	public static final File DEFAULT_SESSION_FILE = new File(RUNELITE_DIR, "session");
+	public static final String DEFAULT_CONFIG_FILE_NAME = "settings.properties";
```

Comment:
I would just remove this completely, as you can use DEFAULT_CONFIG_FILE.getName()